src/OrleansServiceBus/Providers/Streams/EventHub/EventHubCheckpointer.cs

```diff
@@ -22,7 +26,12 @@ public class EventHubCheckpointer : IStreamQueueCheckpointer<string>
         /// <summary>
         /// Indicates if a checkpoint exists
         /// </summary>
-        public bool CheckpointExists => entity != null && entity.Offset != EventHubConsumerGroup.StartOfStream;
+        public bool CheckpointExists => entity != null && entity.Offset !=
+#if NETSTANDARD
+            PartitionReceiver.StartOfStream;
+#else
+            EventHubConsumerGroup.StartOfStream; 
+#endif
```

Comment:
third time I see this, should you create a constant with the conditional compilation, and then just have these different portions of the code just point to our constant? It does make sense logically regardless of the different versions of the nuget, if this is not JUST an opaque constant to us, let's just hold on to it.